src/main/java/io/cryostat/recordings/RecordingHelper.java

```diff
@@ -154,10 +155,7 @@ public ActiveRecording startRecording(
                             boolean restart =
                                     shouldRestartRecording(replace, previousState, recordingName);
                             if (!restart) {
-                                throw new BadRequestException(
-                                        String.format(
-                                                "Recording with name \"%s\" already exists",
-                                                recordingName));
+                                throw new EntityExistsException("Recording", recordingName);
```

Comment:
I guess this should either have a `public static class RecordingExistsException` to use, or else in `Rules.java` it should just `throw new EntityExistsException` like here.